test/services/SubmissionServiceSpec.scala

```diff
@@ -41,156 +42,298 @@ class SubmissionServiceSpec
     with MockSoleTraderMatchingService
     with MockBusinessVerificationService
     with MockCreateTrnService
+    with MockRegistrationOrchestrationService
     with FeatureSwitching {
 
-  object TestService extends SubmissionService(
-    mockJourneyService,
-    mockSoleTraderMatchingService,
-    mockSoleTraderIdentificationService,
-    mockBusinessVerificationService,
-    mockCreateTrnService)
-
   implicit val hc: HeaderCarrier = HeaderCarrier()
 
+  "submit: sole trader journey with businessVerificationCheck = false" when {
+
+    "the user has a sautr" should {
+
+      s"register Without Business Verification" in {
+
+        val journeyConfigWithoutBV = testSoleTraderJourneyConfig.copy(businessVerificationCheck = false)
+
+        mockRetrieveIndividualDetails(testJourneyId)(Future.successful(Some(testIndividualDetails)))
+        mockMatchSoleTraderDetails(testJourneyId, testIndividualDetails, journeyConfigWithoutBV)(Future.successful(Right(true)))
+        mockRegisterWithoutBusinessVerification(testJourneyId, testIndividualDetails.optNino, testIndividualDetails.optSautr.get)(Future.successful(Registered("someId")))
```

Comment:
You can replace "someId" with testSafeId - it's a variable we already have in TestConstants